### PR TITLE
added full calculator functionality

### PR DESCRIPTION
**Pull request by kunmi02**
I added the full calculator functionality

### CODE REVIEW

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes required ♻️ 

**Hello @kunmi02**

Great job with the project so far it looks amazing, but there are still some changes to be made:
 
---
### Requested 🔴 

You did a great job with this milestone. You had a lot of logic errors last milestone but I didn't want to point them out since you haven't actually tried it yet, but you did a great job with fixing these issues. 

- [ ] One last thing, Your calculator should have a Division by zero error when a user tries to divide any number by zero. 
 

---

please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

**Review by taiwo2 — CHANGES_REQUESTED** [review]
# STATUS: CHANGED REQUIRED ♻️ 
@kunmi02 well done on the changes suggested by my previous Code reviewer, however, the suggested changes are not yet implemented in an expected way because when you try to divide Number by Zero should display `Division by zero error` not `0`
kindly see my inline comments to fix it. thanks

### once specific changes are made kindly request for review

File: src/logic/operate.js [hunk 1]
```diff
@@ -1,17 +1,39 @@
...
 const operate = (numberOne, numberTwo, operation) => {
-  const num1 = new Big(numberOne);
-  const num2 = new Big(numberTwo);
+  const calculate = (num1, num2, op) => {
+    switch (op) {
+      case '+': return num1.plus(num2).toFixed();
+      case '*': return num1.times(num2).toFixed();
+      case '/':
+        if (parseInt(num2, 10) === 0) {
+          return '0';
```
Comment (taiwo2) [inline src/logic/operate.js]: should return  `Division by zero error` not `0` because displaying `0` those not means division error

**Review by taiwo2 — APPROVED** [review]
# STATUS: APPROVED 🎉 
@kunmi02 congratulations on completing this milestone requirement, its time to merge 🥂 🥂

- [ OPTIONAL]
- Kindly update your changes to the live demo in your readme

**Pull request merged**